packages/fx-core/resource/package.nls.json

```diff
@@ -691,6 +693,8 @@
   "driver.teamsApp.progressBar.publishTeamsAppStep1": "Checking if the Teams app has already been submitted to tenant App Catalog",
   "driver.teamsApp.progressBar.publishTeamsAppStep2.1": "Update published Teams app",
   "driver.teamsApp.progressBar.publishTeamsAppStep2.2": "Publishing Teams app...",
+  "driver.teamsApp.progressBar.validateWithTestCases": "Submitting validation request...",
+  "driver.teamsApp.progressBar.validateWithTestCases.step": "Validation request has been submitted, status: %s. You will be prompted once the result is ready or you can check all your validation records in [Teams Developer Portal](%s).",
```

Comment:
Proposed: Validation request submitted, status: %s. You'll be notified when the result is ready or you can check all your validation records in [Teams Developer Portal](%s).